jobs/config.json

```diff
@@ -10231,6 +10231,15 @@
       "sig-cluster-lifecycle"
     ]
   },
+  "periodic-kubernetes-e2e-rpms-pushed": {
+    "args": [
+      "./tests/e2e/pinned_rpms.sh"
```

Comment:
can you clarify the requirements for this script? right now it looks like it will need to be run from a container set up with yum and expects some particular configuration file paths, I don't think we currently have an image that covers this.

Reply:
Is this still the case? How much work is required to add an RPM image?

Reply:
bazelbuild installs `rpm` but this runs `yum update -y`, all of our prow images are debian / ubuntu / golang (debian) based AFAIK.

Reply:
Can we just use a Fedora or CentOS image?

Reply:
Ish, we have to add bootstrap etc, look at image/gcloud for a pretty minimal image that supports our checkout and log upload runner etc. In the near future we will (hopefully) be moving this code out of the images but right now the image needs to run `jenkins/bootstrap.py`.

Reply:
Ah I see. So it's just a case of adding a new dir for rpm following gcloud? If so I'll try to submit a PR if I get some cycles soon

Reply:
more or less, someone will also have to keep that image up to date. 
It will need python, gcloud sdk, and that runner setup so we can do the logs / test results upload etc. at the moment. All prow jobs with testgrid/gubernator results use that entrypoint script for now. 
You *could* just run a stock fedora image or something but it wouldn't handle log / artifact / metadata upload at the moment.